### PR TITLE
Include Plugins disable on FAQ

### PR DESCRIPTION
**Pull request by itscynxx**
PR 3/4 to move information from the discord's FAQ channel to the wiki's FAQ.

Honestly, not sure if this is still relevant however I figured better safe than sorry, and it's the last thing in the channel not currently documented/being documented

Requires:
- https://github.com/R2Northstar/NorthstarLauncher/pull/459

### CODE REVIEW

**Comment by catornot** [comment]
I am pretty sure this doesn't exist anymore 

edit: I am talking about the startup arg which was removed in plugins v2

**Comment by GeckoEidechse** [comment]
> I am pretty sure this doesn't exist anymore
> 
> edit: I am talking about the startup arg which was removed in plugins v2

Oh, maybe a CLI arg to prevent plugin would actually be nice tbh ^^

**Comment by catornot** [comment]
ok, I will try to add that and pr it tmr

**Comment by GeckoEidechse** [comment]
Set to draft until the PR the re-implements the feature is merged and released ^^

**Comment by catornot** [comment]
I actually read the changes instead of focusing on `-noplugins` and I just don't understand why this is needed in this form. The default discord rpc plugin doesn't panic normally ( it only panics in extreme cases like mutex getting poisoned or northstar/tf2 giving garbage data ). For any other plugin the author should have probably tested their plugin.

**Comment by GeckoEidechse** [comment]
> I actually read the changes instead of focusing on `-noplugins` and I just don't understand why this is needed in this form.

Given that plugins cannot be disabled like mods I guess it might be useful to quickly disable plugins for testing / find a crashing plugin?

> For any other plugin the author should have probably tested their plugin.

You have to much faith in our modders xD

**Comment by catornot** [comment]
> Given that plugins cannot be disabled like mods I guess it might be useful to quickly disable plugins for testing / find a crashing plugin?

if we want plugins to be able to disabled individually we just need to add a blacklist for loading plugins to the profile folder root like for mods. 
and maybe add an option to dismount a plugin at runtime. to do so we just need to free the dll and hope that the plugin is able to dismount it self since threads might not get terminated and also all the sqvms would have to get reloaded to refresh registered native function.

> You have to much faith in our modders xD

people would just remove the faulty plugin/mod. right?

**Comment by catornot** [comment]
also telling people that they are crashing at the beginning because of plugins and suggesting them to disable them isn't a real solution imo. fixing the plugin would be a better way lol.

**Comment by itscynxx** [comment]
> people would just remove the faulty plugin/mod. right?

Oh, you'd be surprised how many times I've solved "help my game doesn't load", I ask if any new mods have been installed, they say yes, I say to remove it then are surprised when it works XD

But also, this isn't entirely my place to say what should/shouldn't exist for _actual_ Northstar stuff as I do wiki things, but I just made this pr due to the fact that it was the last non documented (or planned to be documented) information from the FAQ discord channel

**Comment by GeckoEidechse** [comment]
https://github.com/R2Northstar/NorthstarLauncher/pull/459 has been merged and released so this should ready for review now, except that it now has a merge conflict :P

**Comment by itscynxx** [comment]
solved merge conflicts

File: docs/faq.md [hunk 1]
```diff
@@ -94,3 +94,8 @@ A: This usually lies behind your computer's specs not being high enough in order
 
 A: Titanfall2 (and other EA games) used to use a program called [Origin](<https://en.wikipedia.org/wiki/Origin_(service)>) to play games, however this has since been replaced by the EA App. 
 This change, however, did not update the game's error messages to say "EA App" instead of "Origin", so any time you see that there's an issue with Origin, you can replace mention of "Origin" with the "EA App".
+
+### Q: When I launch Northstar, a small command prompt appears for a few seconds then closes as nothing else happens!
+
+A: Delete `R2Northstar/plugins`, which disables any currently installed plugins, the only default plugin being Northstar's Discord Rich Presence. Alternatively, you could insert `-noplugins` in your `ns_startup_args.txt` in the [titanfall2 directory](installing-northstar/troubleshooting.md#game-location), which would simply disable plugins instead of deleting them.
```
Comment (GeckoEidechse) [inline docs/faq.md:100]: I would maybe add like a short (single sentence is enough) explanation why this is done.

The whole thing kinda feels like

> \> X isn't working, what do I do?
> \> Just do Y, not gonna tell you why, just do it.

^^

Comment (itscynxx) [reply docs/faq.md:100]: idk why either, its just something that was still left over in #faq, where it wasnt explained either

Comment (GeckoEidechse) [reply docs/faq.md:100]: eh, I guess

**Pull request merged**